zproject/prod_settings_template.py

```diff
@@ -398,7 +401,7 @@
         "url": "{}{}".format("https://", EXTERNAL_HOST),
     },
 }
-SOCIAL_AUTH_SAML_ENABLED_IDPS: Dict[str, Any] = {
+SOCIAL_AUTH_SAML_ENABLED_IDPS: Dict[str, "SAMLIdPConfigDict"] = {
```

Comment:
I think we may want to skip this commit -- `prod_settings_template.py` is not modified when upgrading a Zulip server, so we generally want to avoid importing things that we may want to rename at some later time.